src/features/device-state/routes/fleet-state-get-v3.ts

```diff
@@ -46,6 +48,14 @@ const fleetExpand = {
 	},
 } as const;
 
+export type ExpandedApplicationWithService = OptionsToResponse<
+	Application['Read'],
+	{
+		$expand: typeof fleetExpand;
+	},
+	undefined
+>[number];
```

Comment:
I think you can also just pass a type for the id to show that you will be getting a specific one rather than the array
```suggestion
export type ExpandedApplicationWithService = OptionsToResponse<
	Application['Read'],
	{
		$expand: typeof fleetExpand;
	},
	number
>;
```

Reply:
@Page- These are not the same, from my testing
```ts
export type ExpandedApplicationWithService = OptionsToResponse<
	Application['Read'],
	{
		$expand: typeof fleetExpand;
	},
	number
>;
```

Will be the same as my version above `| undefined` and my understanding is that we don't want it to be nullable. Ofc I can just wrap it in a `NonNullable` but I am not 100% sure which one is best considering this